packages/js/src/MatomoTracker.ts

```diff
@@ -52,6 +61,14 @@ class MatomoTracker {
     }
   }
 
+  static enableHeartBeatTimer(seconds: number) {
+    window._paq.push(['enableHeartBeatTimer', seconds])
+  }
```

Comment:
According to the documentation `enableHeartBeatTimer` can also be enabled without passing in the seconds, we should reflect this as well.

```suggestion
  static enableHeartBeatTimer(seconds?: number) {
    if (seconds !== undefined) {
      window._paq.push(['enableHeartBeatTimer', seconds])
    } else {
      window._paq.push(['enableHeartBeatTimer'])
    }
  }
```